### ISSUE TITLE
torch-nightly `tensor.data.set_(...)` fails in updates

### ISSUE THREAD

**Issue by ferrine** [ferrine/geoopt#21]
Seems like torch nightly has just changed the behaviour for ``tensor.data.set_(...)`` and now raises an error

https://travis-ci.com/ferrine/geoopt/jobs/167379249#L1406

<details>

```
    def proj_(self):
>       self.data.set_(self.manifold.projx(self.data))
E       RuntimeError: set_storage is not allowed on Tensor created from .data or .detach()
```

<details/>

**Comment by ferrine** [comment]
Some discussion well be held there: https://discuss.pytorch.org/t/api-change-for-tensor-data-set-in-torch-nightly/33310

**Comment by ferrine** [comment]
Closed via #22
